arm/Microsoft.Network/privateDnsZones/.parameters/parameters.json

```diff
@@ -15,6 +15,19 @@
                 }
             ]
         },
+        "a": {
+            "value": [
```

Comment:
Also, I'd suggest to add a min parameters file (linked both in GitHub & ADO) while we're at it

Reply:
done [![Network: PrivateDnsZones](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privatednszones.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.privatednszones.yml)